packages/babel-plugin-transform-class-properties/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "babel-plugin-transform-class-properties",
   "version": "6.16.0",
-  "description": "",
+  "description": "This plugin transforms es2015 static class properties as well as properties declared with the es2016 property initializer syntax",
```

Comment:
To ensure this description stays valid as long as possible I would not add any references to ES201X. So that if we add support for private class property initializers the description stays valid.